src/clickhouseEngine.ts

```diff
@@ -159,14 +160,18 @@ export async function clickhouseEngine(): Promise<void> {
         standardTx.status,
         Math.round(standardTx.timestamp),
         standardTx.usdValue,
-        standardTx.indexVersion
+        config.clickhouseIndexVersion
```

Comment:
I thought this wouldn't be needed anymore now that we have the updatetime. Removing this prevents extra writes to couchdb